telco-examples/edge-clusters/aarch64/README.md

```diff
@@ -0,0 +1,116 @@
+# Tech-Preview AARCH64 Integration (full e2e, mgmt-cluster and downstream clusters using aarch64 architecture)
+
+This is a tech-preview example to demonstrate how to deploy an edge cluster for Telco using SUSE ATIP and the fully automated directed network provisioning using aarch64 architecture.
+
+## Introduction
+
+This is an example to demonstrate how to deploy an edge cluster for Telco using SUSE ATIP and the fully automated directed network provisioning using aaarch64 architecture.
+
+There are two steps to deploy an edge cluster:
+
+- Create the image for the edge cluster using the Edge Image Builder in order to prepare all the packages dependencies and the requirements for the edge cluster.
+- Deploy the edge cluster using metal3 and the image created in the previous step.
+
+Important notes:
+
+* In the following examples, we will assume that the management cluster is already deployed and running. If you want to deploy the management cluster, please refer to the [Management Cluster example](../mgmt-cluster).
+* In the following examples, we are assuming that the edge cluster will use Baremetal Servers. If you want to deploy the full workflow using virtual machines, please refer to the [metal3-demo repo](https://github.com/suse-edge/metal3-demo)
+
+## Create the image for the edge cluster
+
+### Prerequisites
+
+Using the example folder `telco-examples/edge-clusters/aarch64/eib`, we will create the basic structure in order to build the image for the edge cluster: 
+
+You need to modify the following values in the `telco-edge-cluster-aarch64.yaml` file:
+
+- `${ROOT_PASSWORD}` - The root password for the management cluster. This could be generated using `openssl passwd -6 PASSWORD` and replacing PASSWORD with the desired password, and then replacing the value in the `telco-edge-cluster.yaml` file.
+- `${SCC_REGISTRATION_CODE}` - The registration code for the SUSE Customer Center for the SLE Micro product. This could be obtained from the SUSE Customer Center and replacing the value in the `telco-edge-cluster.yaml` file.
+
+You need to modify the following folder:
+
+- `base-images` - To include inside the `SL-Micro.aarch64-6.1-Base-RT-GM.raw`  image downloaded from the SUSE Customer Center.
+
+### Building the Edge Cluster Image using EIB
+
+All the following commands in this section could be executed using any linux laptop/server x86_64 with podman installed. You don't need to have a specific environment to build the image.
```

Comment:
Are we sure we want to suggest this? Isn't it better to suggest using aarch64 build host?

Reply:
yeah I agree..Performance is better as I tested. Thanks I've change it